tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
oh, so by default it gona fix only docblock? then cool, let fix case you mention with option=true as well.

as main target would be docblock, i would move it to phpdoc group

we don't split fixers if they are the same except config.